source/__tests__/integration/create-property.test.js

```diff
@@ -43,23 +35,78 @@ describe('Create profile has properties for resource and templates for property'
       //  to show a template can be selected
       await expect_sel_to_exist(`${template_select_sel}.ng-pristine`)
       await expect_sel_to_exist(`${template_select_sel} > option[selected="selected"][value="?"]`)
-      const value_selected = 'profile:bf2:Form'
-      await page.select(template_select_sel, value_selected)
+      await page.select(template_select_sel, 'profile:bf2:Form')
       await expect_sel_to_exist(`${template_select_sel}.ng-dirty`)
       await expect_sel_to_exist(`${template_select_sel} > option[selected="selected"][value^="profile:bf2"]`)
     })
   })
 
-  describe('property header appearence changed', () => {
-
+  describe('property header appearence', () => {
     it('font-awesome icon class is present', async () => {
       await expect_sel_to_exist(`.fa-caret-right`)
     })
+  })
+})
+
+describe('property URI is required', () => {
+  let profileFormSel = 'form[name="profileForm"]'
+  let exportButtonSel = 'a.btn.import-export'
+
+  it('error if exported without property URI', async () => {
+    await page.waitFor(1000)
+    await expect(page).toFillForm(profileFormSel, {
+      id: "my:profile",
+      description: "Profile description",
+      author: "Me",
+      title: "My profile",
+      resourceId: "my:resource",
+      resourceURI: "http://www.example.com"
+    })
+
+    await page.click(exportButtonSel)
+    let alertBoxSel = '#alertBox > div.modal-dialog > div.modal-content > div.modal-body > p#alert_text'
+    await expect_value_in_sel_text(alertBoxSel, 'Parts of the form are invalid')
```

Comment:
FYI, in my branch, which doesn't even include this, I just got this:

```
server/whichrt called for uri other than ontology http://www.example.com
 FAIL  source/__tests__/integration/create-resource.test.js
  ● Create profile resource template requirements › resource template form fields › requires Resource URI

    expect(received).toBe(expected) // Object.is equality

    Expected: "Parts of the form are invalid"
    Received: "my:resource must have at least one property template"

      86 | async function expect_value_in_sel_textContent(sel, value) {
      87 |   const sel_text = await page.$eval(sel, e => e.textContent)
    > 88 |   expect(sel_text).toBe(value)
         |                    ^
      89 | }
      90 | 
```

which is probably the error message we want in this case, no?

Reply:
@ndushay this was a WIP. Sees like there was an issue with the form fills not happening in the correct order and `waitUntil: 'networkidle2'` was the thing that fixed it.